BegeWrite/Converters/CheckListBlockDataConverter.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Text.Json;
+
+namespace BegeWrite.Converters
+{
+    public class CheckListBlockDataConverter : IBlockDataConverter
```

Comment:
These are pretty ugly and complicated, there is probably an easier way.

However it works and we can refactor these later without breaking anything